migrations/173_questions_filename.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE questions ADD COLUMN thumbnail_filename text;
```

Comment:
Will defer to @mwest1066 on this, but can we combine migrations 173 and 174 into one file?